jujugui/static/gui/src/app/components/status/status.js

```diff
@@ -266,11 +361,13 @@ class Status extends React.Component {
           columnSize: 3,
           content: urlParts[1]
         }],
+        filterValue: this._normaliseStatus(app.status.current),
```

Comment:
Could this be named status something? If this is an arbitrary attribute assigned to a row, maybe it could be more explicitly associated with the filter predicate.

Reply:
It needs to be a generic term as this table will be used elsewhere.

Reply:
I understand this, and you can leave it of course. I was thinking about being more generic as the caller defines and provides the filter predicate, so it could decide to use any field she wants.